docs/HelpStyleGuide.md

```diff
@@ -0,0 +1,68 @@
+
+![Help Text](/docs/images/helptext.png)
+
+## How to Store and Name Files
+
+Help text content is stored into the aces/Loris repo as markdown (.md) files within the help/subdirectory of the module it belongs to. For example, the Help text for Imaging Browser main page is located here: `modules/imaging_browser/help/imaging_browser.md`. 
+
+Some modules have more than one Help text file to correspond with subpages. These files exist in the same `help/` folder, but are named differently - they must match exactly the php/file name that spawns the page. For example, the Imaging Browser module has a subpage called “ViewSession”. Its help file is stored as: `modules/imaging_browser/help/viewsession.md`. 
+
+## Markdown
+
+Formatting should follow markdown guidelines, with some exceptions. This is because LORIS’s current markdown template may not cover all possible markdown features (i.e. bullets). Help text should not include HTML tags for security reasons. When creating a markdown file, use only the following styles, or else talk to the LORIS technical team about expanding our Markdown features:
+
+* `*Italics*`
+* `**Bold**`
+* `***Italics and Bold***`
+* `# Heading 1` up to `###### Heading 6`
+  * Use `#H1` for the name of the module at the beginning of the help text
+  * Use `#H2` for any subheadings
+  * Use `#H4` for sub-subheadings
+* Avoid using HTML line break (`<br>`) tags—simply use physical line breaks (spaces between lines) where necessary. 
+  * Using `<br>` might be necessary only in lists of items
+  
+  ## When to Use Emphasis
+  
+Whether a user knows it or not, they will come to recognize styles that are consistent throughout the text. This way, their eyes can jump to a part of the text that addresses their issue, if they don’t need to read the whole thing. Follow these rules on using emphasis:
+
+* *Italicize* when referring to a specific place on the screen
+  * i.e. in the Selection Filters section
+  * i.e. in the Metadata column of the table
+  * i.e. in the Imaging menu
+* **Bold** when referring to any clickable button, including modules and tabs
+  * DON’T call it a button
+    * i.e. Click **Submit** (rather than “click the **Submit** button”)
+  * DO call it a module or tab
+    * i.e. In the **Imaging Browser** module
+    * i.e. Click the **Upload** tab
+  * You can also use bold when defining features in a list, to separate the term being defined from the definition itself, i.e. when describing columns in a table:<br>
+     **Column heading 1**: description<br>
+     **Column heading 2**: description<br>
+     Etc.
+     
+## Tone
+
+LORIS is used by scientific community worldwide, including countries where English is a second, or even third language. Therefore, it is important that LORIS documentation (help text, setup guides, etc.) be written in a way that it comprehensible by people with limited knowledge and understanding of English language and expression.
```

Comment:
To shorten, and fix 2 grammar issues: 
```suggestion
LORIS is used by scientific community members worldwide, which means it's possible that most users are non-native English speakers. Writing our Help text or any LORIS documentation (setup guides, etc.) in a way that it is quickly comprehensible by people with many different levels of English language and expression.
```

Reply:
`... in a way that is quickly ...`

I would just do a little change to the last sentence.

Reply:
Ok thanks - i updated my suggestion to include your change @PapillonMcGill